app/src/main/res/layout/activity_zim_host.xml

```diff
@@ -34,13 +34,13 @@
   <Button
     android:id="@+id/startServerButton"
     android:layout_width="match_parent"
-    android:layout_height="wrap_content"
+    android:layout_height="60dp"
```

Comment:
why this change?

Reply:
because in wrap content the height of the button becomes the default.

Reply:
I just checked in the release version of the app the height of the button is the default. Should I change to wrap_content?

Reply:
yeah, this ticket wasn't about changing the button height

Reply:
Also I should correct the notion of a "default" for height, no such thing exists. There are 4 options `wrap_content`, `match_parent`, the irrationally hated (by me at least) `fill_parent` and specifying a value.
If you don't specify a height then the app will refuse to compile.
Buttons do mandate a `minHeight/Width` in their default styling to comply with the guidelines on touch areas though, if memory serves

Reply:
Should i remove the height?

Reply:
revert your change and leave the height as it was, `wrap_content`